test/test_multi_region_cubed_sphere.jl

```diff
@@ -773,11 +773,6 @@ end
 end
 
 @testset "Testing simulation on conformal and immersed conformal cubed sphere grids" begin
-    for f in readdir(".")
-        if occursin(r"^cubed_sphere_(output|checkpointer)_.*\.jld2$", f)
-            rm(f; force=true)
-        end
-    end
```

Comment:
Hope this removal is okay. The reason for doing this is that this pattern, when executed by the GPU test group, can delete files being used by the CPU test group.